lib/pbench/common/constants.py

```diff
@@ -2,3 +2,6 @@
 DEFAULT_PBENCH_AGENT_INSTALL_DIR = "/opt/pbench-agent"
 DEFAULT_SCP_OPTS = "-o StrictHostKeyChecking=no"
 DEFAULT_SSH_OPTS = DEFAULT_SCP_OPTS
+DEFAULT_PROM_REG = "quay.io/prometheus/prometheus"
+DEFAULT_PMLOGGER_REG = "quay.io/meyceoz/pcp-collector"
+DEFAULT_PMCD_REG = "quay.io/meyceoz/pcp-exposer"
```

Comment:
`s/meyceoz/pbench/`, and then add the Dockerfile for this collector to our `agent/containers/images` directory?